openapi.yaml

```diff
@@ -9027,12 +9068,18 @@ paths:
           curl -H "Content-Type: application/json" \
             -H "Authorization: Bearer $TOKEN" \
               -X POST -d '{
-                "region": "us-central"
+                "region": "us-central",
+                "placement_group": {
+                  "id": 528,
+                  "compliant_only": true
+                }
               }' \
               https://api.linode.com/v4/linode/instances/123/migrate
       - lang: CLI
         source: >
-          linode-cli linodes migrate 123 --region us-central
+          linode-cli linodes migrate 123 \
+          --region us-central \
+          --placement_group '{"id": 528, "compliant_only": true}' \
```

Comment:
No support for "compliant_only" yet. Need to change this to:

`--placement_group.id 528 \`